web-client/src/components/Modals/ChangeNameModal.tsx

```diff
@@ -0,0 +1,113 @@
+import { UserOutlined } from '@ant-design/icons/lib';
```

Comment:
The code here is very good but I feel you are not coding to the proper level of abstraction.
This code is very specific to a single use case.

The existing Modals have several use cases:
(1) ExternalTriggered full-page modals
(2) Modals that occur once and then a token is saved to local storage.

The components directory is for general-usage components.  

This is too specific.  

Write the general usage file.  Then figure out how you can pass in content or children.  Look at how ExternalTriggered modal works -- you just pass in any markup.  Onetime Modal takes an array of strings as props. 

Figure out the right implementation